### ISSUE TITLE
Obsolete dates should be ISO-8601

### ISSUE THREAD

**Issue by CharlesNepote** [openfoodfacts/openfoodfacts-server#8949]
Currently, dates related to obsolete products are stored as is:
```json
    "obsolete_since_date": "2023/07/01",
    "obsolete_since_date_imported": "2023/07/01"
```
([Example](https://world.openfoodfacts.org/api/v0/product/3596710131563.json).)

Since a very long time in Open Food Facts we use [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) international date format, which tends to be a common international standard in many fields (data science, research, economy, etc.). The dates should be written: ```2023-07-01```.

**Comment by rajm-dv** [comment]
@CharlesNepote Can I work on this issue?

**Comment by CharlesNepote** [comment]
@rajm-dv yes! Sorry for my late reply. I don't know if it's easy to do or not.

Obsolete products are described here: https://wiki.openfoodfacts.org/Obsolete_products
